lfucg-exactions/client/js/components/LoginPage.js

```diff
@@ -102,8 +94,13 @@ function mapDispatchToProps(dispatch) {
         onLogin(event) {
             event.preventDefault();
             dispatch(login())
-            .then(() => {
```

Comment:
This was being used to return the user to the page they were on when they clicked Login.

Reply:
It's going to redirect them directly back to the reset password page unless we find a better option.